docker/colony-cdapp-dev-env-orchestration.yaml

```diff
@@ -88,7 +88,7 @@ services:
     image: colony-cdapp-dev-env/base:latest
     volumes:
       - 'amplify-mock-data:/mock-data'
-    command: "npx --yes http-server --cors --port 3006 /mock-data/colonyNetworkArtifacts"
+    command: 'npx --yes http-server --cors --port 3006 /mock-data/colonyNetworkArtifacts'
```

Comment:
Unnecessary change?

Reply:
It was bugging me that the rest of the strings in this yaml file are single-quoted 😕 Should I revert it @iamsamgibbs?

Reply:
Oh that is fine with me! My brain saw these the wrong way round and thought you were making it not match. 🙃